VORP-Housing/VORP.Housing.Server/Init.cs

```diff
@@ -0,0 +1,315 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using VORP.Housing.Server.Extensions;
+using VORP.Housing.Server.Scripts;
+using VORP.Housing.Shared;
+using VORP.Housing.Shared.Models;
+using VORP.Housing.Shared.Models.Json;
+
+namespace VORP.Housing.Server
+{
+    public class Init : Manager
```

Comment:
Would name the class more around what it is doing, I understand its an "Init" but an initiation of what?

Reply:
To be real with you, I haven't figured out a decent name to replace the old `vorphousing_cl_init` or `vorphousing_sv_init` classes

Reply:
I guess I could call it `InitHousing`, but I'm open to suggestions

Reply:
Does look that the class is only dealing with house events, the resource naming makes it more confusing.